x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -33,27 +33,58 @@
       ,"ignore_missing": true
       }
     }
+  , {"lowercase":
+       {"field": "suricata.eve.http.http_method"
+       ,"target_field": "http.request.method"
+       ,"ignore_missing": true
+      }
+    }
   , {"convert":
-      {"field": "suricata.eve.http.http_method"
-      ,"target_field": "http.request.method"
+      {"field": "suricata.eve.http.status"
+      ,"target_field": "http.response.status_code"
       ,"type": "string"
       ,"ignore_missing": true
       }
     }
   , {"convert":
-      {"field": "suricata.eve.http.status"
-      ,"target_field": "http.response.status_code"
+      {"field": "suricata.eve.http.hostname"
+      ,"target_field": "url.domain"
       ,"type": "string"
       ,"ignore_missing": true
       }
     }
+  , { "grok":
+      { "field": "suricata.eve.http.url"
+      , "patterns": ["%{PATH:url.path}(?:\\?%{QUERY:url.query})?(?:#%{ANY:url.fragment})?"]
+      , "ignore_missing": true
+      , "pattern_definitions":
+        { "PATH": "[^?#]*"
+        , "QUERY": "[^#]*"
+        , "ANY": ".*"
+        }
```

Comment:
Thanks for moving to grok here ❤️ The golden files haven't been re-generated, so it's hard to validate. It's also why CI is failing right now :-)